pulsar/test_helper.go

```diff
@@ -153,15 +153,24 @@ func deleteTopic(topic string) error {
 
 func topicStats(topic string) (map[string]interface{}, error) {
 	var metadata map[string]interface{}
-	err := httpGet("admin/v2/persistent/"+topicPath(topic)+"/stats", &metadata)
+	tp, err := topicPath(topic)
+	if err != nil {
```

Comment:
Not sure if this is needed since it's a test helper method. The test should fail without this check if a bad topic name is passed. Same for the topicPath function.

Reply:
This is just a protective measure to prevent modification of test cases. It's just that I will do it. I'm not sure that others are the same